diff --git a/node_modules/@wdio/appium-service/node_modules/webdriver/build/node.js b/node_modules/@wdio/appium-service/node_modules/webdriver/build/node.js
index eab68a3..2fce41c 100644
--- a/node_modules/@wdio/appium-service/node_modules/webdriver/build/node.js
+++ b/node_modules/@wdio/appium-service/node_modules/webdriver/build/node.js
@@ -2269,7 +2269,7 @@ var FetchRequest = class extends WebDriverRequest {
     try {
       const globalDispatcher = getGlobalDispatcher();
       if (globalDispatcher && (globalDispatcher.constructor.name === "ProxyAgent" || "proxy" in globalDispatcher || // Also check for other custom dispatcher types
-      globalDispatcher.constructor.name !== "Agent" && globalDispatcher.constructor.name !== "MockAgent")) {
+      globalDispatcher.constructor.name !== "Agent" && globalDispatcher.constructor.name !== "MockAgent" && !globalDispatcher.constructor.name.endsWith("Wrapper"))) {
         return globalDispatcher;
       }
     } catch {
